lib/handlers/get-unimind/handler.ts

```diff
@@ -129,4 +142,83 @@ export class GetUnimindHandler extends APIGLambdaHandler<ContainerInjected, Requ
     const getUnimindRequestByPairMetricName = `GetUnimindRequestPair${pair}`
     metrics.putMetric(getUnimindRequestByPairMetricName, 1, Unit.Count)
   }
-} 
\ No newline at end of file
+}
+
+export function artemisModifyCalldata(calldata: string, log: Logger): string {
+    try {
+      const functionSignatures: Record<string, string> = {
+        "24856bc3": "function execute(bytes commands, bytes[] inputs)",
+        "3593564c": "function execute(bytes commands, bytes[] inputs, uint256 deadline)"
+      };
+
+      const functionSelector = calldata.slice(2, 10)
```

Comment:
Can they be derived in the code to make it explicit? Something like:
```suggestion
      const prefix = "0x"
      const functionSelectorBytes = 4
      const charsPerByte = 2
      const functionSelector = calldata.slice(prefix.length, prefix.length + functionSelectorBytes * charsPerByte)
```